tests/test_cases.yaml

```diff
@@ -136,7 +136,20 @@ tests:
       stderr:
         - 'unknown command "non-existent" for "atmos"'
       exit_code: 1
-
+  - name: atmos validate editorconfig
```

Comment:
Great

Reply:
Maybe add one check that verifies the negative.

Reply:
Added one :)